src/main/resources/db/changelog/ddl/0018_EIP1-4481_alter_photo_location_arn_column_in_tables.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018a_EIP1-4481_add_photo_location_arn_to_certificate_table - add photo_location_arn column" context="ddl">
+        <addColumn tableName="certificate">
+            <column name="photo_location_arn" type="varchar(1024)" afterColumn="status">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+
+        <rollback>
+            <dropColumn tableName="certificate" columnName="photo_location_arn"/>
+        </rollback>
+    </changeSet>
+
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018b_EIP1-4481_certificate_table_data_migration - populate photo_location_arn column of certificate">
+        <sql>
+            <comment>Populating [photo_location_arn] of a certificate entity from any of the matching print_requests [photo_location_arn]</comment>
+            UPDATE certificate cert
+            INNER JOIN print_request pr on cert.id = pr.certificate_id
+            SET cert.photo_location_arn = pr.photo_location_arn
+            WHERE cert.photo_location_arn IS NULL
+        </sql>
+
+        <rollback/>
+    </changeSet>
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018c_EIP1-4481_remove_photo_location_arn_column_in_print_request_table - Drop photo_location_arn column" context="ddl">
+        <dropColumn tableName="print_request" columnName="photo_location_arn"/>
+
+        <rollback>
+            <addColumn tableName="print_request">
+                <column name="photo_location_arn" type="varchar(255)" afterColumn="supporting_information_format">
+                    <constraints nullable="false"/>
```

Comment:
This would fail for existing columns - I think would need to add column, copy data from the certificate table...

Reply:
Yeah you are right, but I thought about this as well, but if it ever came to this rollback, then I expected to investigate  and add few more liquibase scripts in new PR. but you are right.

Reply:
Possibly, we'd have to see what state the database is in and see what the appropriate rollback is.  In any case should be able to review the Liquibase generate rollback before it's applied.

Reply:
(The rollback statement of removing from print_request has been changed
- Removes the column from print_request
- If failed (i.e. rollback) then add new nullable column, copy the data from certificate table and then make print_request column back to non-nullable.